index.html

```diff
@@ -296,4 +297,125 @@ <h2>About me</h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+    </div>
+    <form
+      action="https://formspree.io/f/xwkyvenp"
+      method="get"
+      class="form"
+      id="form"
+    >
+      <div class="name">
+        <label for="name"></label>
+        <input
+          type="text"
+          name="fullName"
+          id="name"
+          placeholder="Full name"
+          required
+          minlength="2"
+          maxlength="20"
+        />
+      </div>
+      <div class="email">
+        <label for="mail"></label>
+        <input
+          type="email"
+          name="mail"
+          id="mail"
+          placeholder="Email"
+          required
+        />
+      </div>
+      <div class="message">
+        <label for="message"></label>
+        <textarea
+          name="message"
+          id="message"
+          placeholder="Enter text here"
+          required
+          maxlength="300"
+          minlength="10"
+        ></textarea>
+      </div>
+      <p id="error-msg"></p>
+      <button type="submit" class="form-submit">Get in touch</button>
+
+    </form>
+    <hr>
+      <div class="social-footer">
+        <a
+          class="social-link"
+          href="https://github.com/SpaYco"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <img
+            class="social-icon-footer"
+            src="./media/png/github.png"
+            alt="GitHub Icon"
+          />
+        </a>
+        <a
+          class="social-link"
+          href="https://www.linkedin.com/in/spayco"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <img
+            class="social-icon-footer"
+            src="./media/png/linkedin.png"
+            alt="LinkedIn Icon"
+          />
+        </a>
+        <a
+          class="social-link"
+          href="https://angel.co/spayco"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <img
+            class="social-icon-footer"
+            src="./media/png/angel.png"
+            alt="AngelList Icon"
+          />
+        </a>
+        <a
+          class="social-link"
+          href="https://twitter.com/iSpaYco"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <img
+            class="social-icon-footer"
+            src="./media/png/twitter.png"
+            alt="Twitter Icon"
+          />
+        </a>
+        <a
+          class="social-link"
+          href="https://spayco.medium.com/"
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          <img
+            class="social-icon-footer"
+            src="./media/png/medium.png"
+            alt="Medium Icon"
+          />
+        </a>
+      </div>
+      <img
+            class="bar-icon-footer"
+            src="./media/png/Shape.png"
+            alt="Medium Icon"
+          />
+  </footer>
```

Comment:
[ ] kindly  implement the buttons interactions, as it's described in the [figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)